src/qix.js

```diff
@@ -20,13 +20,14 @@ class Qix {
     const {
       url,
       createSocket,
-      delta,
+      protocol,
       Promise,
       responseInterceptors,
       JSONPatch,
       definition,
     } = config;
-    const apis = new ApiCache({ Promise, definition });
+    const delta = protocol.delta;
```

Comment:
I agree.
However, `protocol.delta` still needs special handling (some some methods are delta blacklisted). Will see how it can be solved.